tests/conftest.py

```diff
@@ -0,0 +1,159 @@
+from pathlib import Path
```

Comment:
Let's rename the file to `ibs_conftest.py`

Reply:
Done